### PR TITLE
DCS-905 added request remover controller, tests and frontend view

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: integration-tests/mockApis/auth.js [hunk 1]
```diff
@@ -86,7 +99,7 @@ const token = ({ userName = 'TEST_USER', isReviewer = false, isCoordinator = fal
         Location: 'http://localhost:3007/login/callback?code=codexxxx&state=stateyyyy',
       },
       jsonBody: {
-        access_token: createToken(isReviewer, isCoordinator),
+        access_token: createClientCredsToken(isReviewer, isCoordinator),
```
Comment (andrewrlee) [inline integration-tests/mockApis/auth.js:102]: This should be `createToken`

File: integration-tests/mockApis/auth.js [hunk 2]
```diff
@@ -97,6 +110,28 @@ const token = ({ userName = 'TEST_USER', isReviewer = false, isCoordinator = fal
...
+      method: 'POST',
+      urlPattern: '/auth/oauth/token',
+    },
+    response: {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json;charset=UTF-8',
+      },
+      jsonBody: {
+        access_token: createToken(),
```
Comment (andrewrlee) [inline integration-tests/mockApis/auth.js:125]: This should be `createClientCredsToken`

File: integration-tests/mockApis/auth.js (hunk 1 above)
Comment (louisenorris) [reply integration-tests/mockApis/auth.js]: Sorted

File: integration-tests/mockApis/auth.js (hunk 2 above)
Comment (louisenorris) [reply integration-tests/mockApis/auth.js]: Now sorted

**Review by andrewrlee — COMMENTED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**